src/app/open-absences/components/open-absences-detail/open-absences-detail.component.ts

```diff
@@ -37,15 +41,41 @@ export class OpenAbsencesDetailComponent
     this.selectionService.selection$,
   ]).pipe(map(([absences, selection]) => absences.length === selection.length));
 
+  studentEmail$ = this.absences$.pipe(
+    map((absences) => (absences[0] && absences[0].StudentRef.Id) || null),
+    switchMap((id) =>
+      id ? this.personService.getByIdWithEmailInfos(id) : EMPTY
+    )
+  );
+  translation$ = this.studentEmail$.pipe(
+    switchMap((person) =>
+      this.translate.getTranslation(
```

Comment:
Ich denke das ist ok so und ich finde das gut. Alternativ müsste man wohl eine Lib wie z.b. https://www.npmjs.com/package/ngx-translate-in einsetzen.

Reply:
Ok, dann lass ich das mal so.
Sollten wir dieses Feature dann mal in verschiedenen Modulen brauchen, können wir ja nochmals darüber diskutieren, ob eine Library nun Sinn macht.